pkg/wwan/Dockerfile

```diff
@@ -3,15 +3,15 @@ FROM lfedge/eve-alpine:9fb9b9cbf7d90066a70e4704d04a6fe248ff52bb as build
 ENV BUILD_PKGS automake autoconf gettext gettext-dev git pkgconfig \
                libtool libc-dev linux-headers gcc make glib-dev \
                autoconf-archive patch cmake gtk-doc
-ENV PKGS alpine-baselayout musl-utils ppp jq glib
+ENV PKGS alpine-baselayout musl-utils ppp jq glib openssl
```

Comment:
We currently only have openssl in the (in theory optional) debug package. It would be nice if we didn't need to include it due to its size/complexity and resulting number of CVEs over time.

Reply:
I use openssl only to decrypt password written to `/run/wwan/config.json` by NIM. This file is mapped to memory so strictly speaking the password could be there in clear-text. However, it will be very easy to accidentally expose this password - content of `/run/wwan` is included in netdumps, also in tar archives created by `collect-info.sh` and customers/CS tend to attach those `/run/wwan/*.json` files to customer issue tickets (because we asked many times for them and they learned the purpose of these JSON files). Therefore I wanted to make sure that the password is unreadable even if `config.json` is leaked. Maybe this is overly paranoid? I'm not an expert on security and don't really have strong opinion on this. If we are OK with moving the user password from NIM to wwan in cleartext than we will not need to add openssl or go-openssl.

Reply:
Yes, but dealing with CVEs in the openssl package is also risky, which is why we try very hard to not use it (and instead use the golang crypto code). How large would a golang standalone binary be which can do the decrypt so we can avoid adding openssl?

Reply:
I will prepare the binary and let you know.
I will also check how hard is it to avoid using `github.com/Luzifer/go-openssl/v4` - after all only one function from openssl is used and maybe it is not so difficult to do the same using just the crypto library.

Reply:
In comparison, the openssl package is a few hundred kb I think.

Reply:
When the EVE image is fully built, the size increase is a bit smaller thanks to the compression.
- Current EVE image size: 311428608 bytes
- With openssl package: 311432704 bytes (current size + 4KB)
- With Go binary instead of openssl: 311916032 bytes (current size + 476KB)

Reply:
How ad-hoc would the checks be to remove then in the third case?
It feelds like the second option is the best path forward.

Reply:
It would be quite ad-hoc, just for this specific case...
I have implemented the second option. The changes are pushed as a separate commit for easier review. Later when approved, I would squash the commits.

Reply:
Thanks